changelog/upgrade-notes.md

```diff
@@ -31,6 +31,12 @@ but with additional css:
 }
 ```
 
+#### Removal of `ol.source.TileVector`
+
+With the introduction of true vector tile support, `ol.source.TileVector` becomes obsolete. To continue rendering tiled vector data with a vector layer, use `ol.source.Vector` with an `ol.loadingstrategy.tile` strategy and `uniqueFeatureIds` set to `false` instead. You also have to clear the source when the map resolution changes. See the [tile-vector example](http://openlayers.org/en/v3.11.0/examples/tile-vector-html) for details.
+
```

Comment:
Still curious to hear your thoughts on when someone would actually do this, instead of using ol.layer.VectorTile and ol.source.VectorTile. Is this not so much of an edge case that it could be removed altogether? If I read these notes as an end user, I would not be able to make my choice.